packages/gestalt/src/Touchable.js

```diff
@@ -25,8 +31,10 @@ type Props = {|
   accessibilityLabel?: string,
   children?: React.Node,
   disabled?: boolean,
+  forwardedRef?: React.Ref<'div'>,
   fullHeight?: boolean,
   fullWidth?: boolean,
+  inline?: boolean,
```

Comment:
i wonder if we actually need this as a prop? this seems like something we should be able to achieve via a container instead of setting it explicitly on Touchable

Reply:
good catch! I can just wrap it with `<Box display="inlineBlock">`. will remove.